cartographer_turtlebot/configuration_files/turtlebot_depth_camera_2d.lua

```diff
@@ -37,8 +37,8 @@ options = {
 MAP_BUILDER.use_trajectory_builder_2d = true
 
 TRAJECTORY_BUILDER_2D.laser_min_range = 0.1,
```

Comment:
Sure enough! The same issue appears in lines 39-41 of turtlebot_urg_lidar_2d.lua  

Would you like me to submit those changes?